### PR TITLE
feat: invite single email

### PR DESCRIPTION
**Pull request by ZeRego**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #2208 

### Preview:

Screenshot 2022-05-24 at 16 20 22
Screenshot 2022-05-24 at 16 20 43
Screenshot 2022-05-24 at 15 11 37

### Scope of the changes (select all that apply):
- [x] Frontend  
- [x] Backend  
- [x] Common  
- [x] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/hooks/useInviteLink.tsx [hunk 1]
```diff
@@ -51,23 +53,26 @@ export const useInviteLink = (inviteCode: string) =>
...
     const { showToastError, showToastSuccess } = useApp();
-    return useMutation<InviteLink, ApiError>(createInviteWith3DayExpiryQuery, {
-        mutationKey: ['invite_link'],
-        onError: (error1) => {
-            const [title, ...rest] = error1.error.message.split('\n');
-            showToastError({
-                title,
-                subtitle: rest.join('\n'),
-            });
+    return useMutation<InviteLink, ApiError, string>(
```
Comment (ZeRego) [inline packages/frontend/src/hooks/useInviteLink.tsx:56]: All I added was a `, string` in this line. The rest of the changes are formatting.

File: packages/frontend/src/components/ReactHookForm/InputWrapper.tsx [hunk 2]
```diff
@@ -46,7 +48,7 @@ const InputWrapper: FC<InputWrapperProps> = ({
 
     return (
         <FormGroup
-            className="input-wrapper"
+            className={`input-wrapper ${className}`}
```
Comment (ZeRego) [inline packages/frontend/src/components/ReactHookForm/InputWrapper.tsx:51]: This change allows styled components lib to apply classnames to our form inputs.

**Comment by rephus** [comment]
😄 nice 
image

**Comment by rephus** [comment]
After adding the user, and registering in another window, I had to refresh the page to see the new user, do you think this a problem ? 
image

**Review by rephus — APPROVED** [review]
Code looks good and works as expected. We just need to fix tests.

**Comment by ZeRego** [comment]
> You can't see all pending invitations anywhere, is this ok ? At leat we can add multiple invitations for the same email without revoking anything.

@rephus We broke down all the new invite work into multiple PR's. This is the first one out of 5 ( or so ). The work to show pending invites/users will be done in this ticket https://github.com/lightdash/lightdash/issues/2184

**Pull request merged**